common/lib/container-definitions/src/runtime.ts

```diff
@@ -121,7 +121,7 @@ export interface IContainerContext extends IDisposable {
     readonly quorum: IQuorum;
     readonly audience: IAudience | undefined;
     readonly loader: ILoader;
-    readonly logger: ITelemetryBaseLogger;
+    readonly logger: ITelemetryLogger;
```

Comment:
@markfields is this correct?

Reply:
Nope, ITelemetryBaseLogger is correct one - all compat boundaries should be using simpler and more dump ITelemetryBaseLogger interface.